app/implicits/ImplicitDateFormatter.scala

```diff
@@ -37,32 +46,29 @@ trait ImplicitDateFormatter {
   implicit class longDate(d: LocalDate)(implicit messages: Messages) {
 
     def toLongDateNoYear: String = {
-      val dt = Dates.formatDate(org.joda.time.LocalDate.parse(d.toString))(messages)
+      val dt = languageUtils.Dates.formatDate(org.joda.time.LocalDate.parse(d.toString))(messages)
       dt.split(" ")(0) + " " + dt.split(" ")(1)
     }
 
     def toLongDateShort: String = {
-      Dates.formatDateAbbrMonth(org.joda.time.LocalDate.parse(d.toString))(messages)
+      languageUtils.Dates.formatDateAbbrMonth(org.joda.time.LocalDate.parse(d.toString))(messages)
     }
 
     def toLongDate: String = {
-      Dates.formatDate(org.joda.time.LocalDate.parse(d.toString))(messages)
+      languageUtils.Dates.formatDate(org.joda.time.LocalDate.parse(d.toString))(messages)
     }
   }
 
   implicit class longDateTime(dt: LocalDateTime)(implicit messages: Messages) {
     def toLongDateTime:String = {
-      Dates.formatDate(org.joda.time.LocalDate.parse(dt.toLocalDate.toString))(messages)
+      languageUtils.Dates.formatDate(org.joda.time.LocalDate.parse(dt.toLocalDate.toString))(messages)
     }
   }
 
   implicit def toLocalDate(s: String): LocalDate = localDate(s).toLocalDate
 
 }
 
-object ImplicitDateFormatter extends ImplicitDateFormatter {
-  implicit val localDateOrdering: Ordering[LocalDate] = Ordering.fromLessThan(_ isBefore _)
-}
 
```

Comment:
can you please only keep row 71 and remove all extra rows after that, just to make Joe happy.